### PR TITLE
LOOOOPING

### PR DESCRIPTION
**Pull request by nivenreed**

### CODE REVIEW

File: data/gallery.js [hunk 1]
```diff
@@ -37,6 +37,40 @@ myAlbum.appendChild(description);
...
+// album.images.forEach(function(myAlbums) {
+//     document.createElement('li');
+//     console.log(myAlbums);
+// }
+// my)
+function logImages(images){
+    const imagesLi = document.createElement('li');
+      imagesLi.textContent = images.title;
+    const imagesImg = document.createElement('img');
+      imagesImg.src = images.src;
```
Comment (openbook) [inline data/gallery.js:72]: You should get into the habit of adding an 'alt' tag to images (it's required for accessibility), so here you could use the image title as the alt tag.

**Review by openbook — CHANGES_REQUESTED** [review]
SICK LOOPS BRO

but, add some altz.

File: data/gallery.js (hunk 1 above)
Comment (nivenreed) [reply data/gallery.js:72]: I did think this. I will add them in.

**Review by openbook — APPROVED** [review]
👌

**Pull request merged**